src/fnn/fnn.ml

```diff
@@ -527,7 +527,7 @@ module Model = struct
       (predict Session.Input.float Session.Output.float : (float, b) Tensor.t)
     | Node.Type.Double, Tensor.Double ->
       (predict Session.Input.double Session.Output.double : (float, b) Tensor.t)
-    | _ -> assert false
+    | _ -> .
```

Comment:
According to the manual, refutation cases were introduced in 4.03. I haven't tested anything with 4.03